src/main/java/com/j256/ormlite/android/annotations/OrmLiteAnnotationProcessor.java

```
@@ -0,0 +1,812 @@
+				// Inability to read is not an error, just initialize with empty
+				// contents
+				savedDatabaseInfo = new HashMap<String, List<String>>();
+			} catch (IOException e) {
+				// Inability to read is not an error, just initialize with empty
+				// contents
+				savedDatabaseInfo = new HashMap<String, List<String>>();
+			} catch (ClassNotFoundException e) {
+				// Built-in Java classes will always be available
+				throw new RuntimeException(e);
+			}
+
+			// Verify each Database annotation only contains valid tables and
+			// add it to the saved list for future rounds of incremental
+			// compilation
+			for (Entry<TypeElement, List<TypeElement>> databaseAndTables : foundDatabases
+					.entrySet()) {
+				List<String> tableNames = new ArrayList<String>();
+
+				for (TypeElement table : databaseAndTables.getValue()) {
+					if (table.getAnnotation(DatabaseTable.class) == null) {
+						raiseError(
+								String.format(
+										"%s annotation contains class %s not annotated with %s",
+										Database.class.getSimpleName(),
+										table.getSimpleName(),
+										DatabaseTable.class.getSimpleName()),
+								databaseAndTables.getKey());
+					}
+
+					tableNames.add(table.getQualifiedName().toString());
+				}
+
+				savedDatabaseInfo.put(databaseAndTables.getKey()
+						.getQualifiedName().toString(), tableNames);
+			}
+
+			// Save the updated information for future rounds of incremental
+			// compilation
+			try {
+				FileObject file = processingEnv.getFiler()
+						.createResource(savedDatabaseInfoLocation,
+								savedDatabaseInfoPackageName,
+								savedDatabaseInfoFileName);
+				ObjectOutputStream writer = new ObjectOutputStream(
+						file.openOutputStream());
+				writer.writeObject(savedDatabaseInfo);
+				writer.close();
+				raiseNote(String.format(
+						"Stored %d Database-to-DatabaseTable mappings",
+						savedDatabaseInfo.size()));
+			} catch (FilerException e) {
+				// intentionally ignore the error
```

Those errors should be logged at the very least. Don't leave empty catch blocks.